tests/Benchmarks/AzureBench.cs

```diff
@@ -15,7 +15,7 @@
 using System.Text.Encodings.Web.Utf8;
 using System.Text.Utf8;
 
-public class AzureBench
+public class CosmosDbBench
```

Comment:
Why not move these to the Azure Test project and add to the perf harness so you get the granularity of running just these performance tests (similar to others like [System.Binary.Base64.Tests](https://github.com/dotnet/corefxlab/blob/master/tests/System.Binary.Base64.Tests/PerformanceTests.cs))?

Reply:
run.bat StorageBench runs just storage tests. i.e. good enough granularity.